array/removeAt.test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import removeAt from "./removeAt.ts.js";
```

Comment:
The import is broken and makes the checks to fail:

```suggestion
import removeAt from "./removeAt.ts";
```